sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -454,3 +454,14 @@ case class ShowTableProperties(
     AttributeReference("key", StringType, nullable = false)(),
     AttributeReference("value", StringType, nullable = false)())
 }
+
+/**
+ * The logical plan of the COMMENT ON (DATABASE|SCHEMA|NAMESPACE) ... IS ... command that works for
+ * v2 catalogs.
+ */
+case class CommentOnNamespace(namespace: Seq[String], comment: String) extends Command
```

Comment:
Can you leave some comments about NULL? just like `NULL to drop the comment`;
https://www.postgresql.org/docs/12/sql-comment.html
```
// From the PgSQL document:
text
   The new comment, written as a string literal; or NULL to drop the comment.
```

Reply:
all right.

Reply:
Thanks!